api/models/trial.js

```diff
@@ -106,6 +107,9 @@ const Trial = BaseModel.extend({
   records: function () {
     return this.hasMany('Record');
   },
+  risks_of_bias: function () {
+    return this.hasMany('RiskOfBias');
+  },
```

Comment:
Need to add the `risk_of_bias` and `risk_of_bias.risk_of_bias_criteria` to the `relatedModels` here. Also, please write a few tests validating that it's working as expected. They can be simple.